tanner/session_analyzer.py

```diff
@@ -99,18 +99,15 @@ def __init__(self, loop=None):
         owner_names = ['user', 'tool', 'crawler', 'attacker']
         possible_owners = {k: 0.0 for k in owner_names}
         attacks = {'rfi', 'sqli', 'lfi', 'xss'}
-        bots_owner = ['Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)',
-                      'Googlebot/2.1 (+http://www.google.com/bot.html)',
-                      'Mozilla/5.0 (compatible; bingbot/2.0; +http://www.bing.com/bingbot.htm)',
-                      'Mozilla/5.0 (iPhone; CPU iPhone OS 7_0 like Mac OS X) AppleWebKit/537.51.1 '
-                      '(KHTML, like Gecko) Version/7.0 Mobile/11A465 Safari/9537.53 (compatible; '
-                      'bingbot/2.0; +http://www.bing.com/bingbot.htm)',
-                      'Mozilla/5.0 (Windows Phone 8.1; ARM; Trident/7.0; Touch; rv:11.0; '
-                      'IEMobile/11.0; NOKIA; Lumia 530) like Gecko (compatible; bingbot/2.0; '
-                      '+http://www.bing.com/bingbot.htm)']
-        possible_owners['crawler'], possible_owners['tool'] = await self.detect_crawler(stats, bots_owner)
-        possible_owners['attacker'] = await self.detect_attacker(stats, bots_owner, attacks)
-
+        with open("tanner/data/crawler_user_agents.txt") as f:
```

Comment:
don't use relative paths, since we can run tanner from any place, it can cause the exception